src/pages/comparison/components/characteristics/characteristics.component.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
-import { ComparisonElement } from './comparison-element.component';
-import '../comparison.scss';
-import { CharacteristicProps, PetToCompare } from '../interfaces/pet.interface';
+import { ComparisonElement } from '../comparison-element/comparison-element.component';
+import '../../comparison.scss';
```

Comment:
Doesn't look like CSS modules :) Take a look at this part

Reply:
Agree with you